buildscripts/semantic-convention/templates/SemanticAttributes.java.j2

```diff
@@ -89,7 +89,7 @@ public final class {{class}} {
   public static final class {{class_name}} {
     {%- for member in attribute.attr_type.members %}
       /** {% filter escape %}{{member.brief | to_doc_brief}}.{% endfilter %} */
-      public static final {{ type }} {{ member.member_id | to_const_name }} = {{ print_value(type, member.value) }};
+      public static final {{ type }} {{ member.member_id | to_const_name | regex_replace(" ", "_") | regex_replace("^([0-9])","_$1") }} = {{ print_value(type, member.value) }};
```

Comment:
Let's just add file an issue to revert this in 1.7.0

Reply:
1.7.0 is going to break a bunch of stuff that went in in 1.6.0, due to this stuff. That's why I was wondering if we should skip 1.6.0. I'm fine either way.

Reply:
Oh I read the linked issue and get it better now - so the `NetHostConnectionSubtypeValues` would change from what I understand. Hrm - I guess we can still go for it, it's what we have the schema URLs for for better or worse

Reply:
I wonder if they'll create a schema for 1.7.0 that maps between the changes. And, how that will interact with the fact that we're having to actually change the keys here, since they aren't java compliant constant values.

Reply:
I think a schema can only handle the values.

Reply:
I guess that makes sense, yes. 👍🏽